lib/features/modeling/behavior/MessageFlowBehavior.js

```diff
@@ -0,0 +1,89 @@
+import inherits from 'inherits';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+import { is } from '../../../util/ModelUtil';
+
+import { isExpanded } from '../../../util/DiUtil';
+
+import { selfAndAllChildren } from 'diagram-js/lib/util/Elements';
+
+import {
+  getResizedSourceAnchor,
+  getResizedTargetAnchor
+} from 'diagram-js/lib/features/modeling/cmd/helper/AnchorsHelper';
+
+/**
+ * BPMN-specific message flow behavior.
+ */
+export default function RemoveParticipantBehavior(eventBus, modeling) {

```

Comment:
Can we rename it to `MessageFlowBehavior`, too? Also, in terms of what it does, why is it not called `ReconnectMessageFlowBehavior`?

I think we're 98.12% to finishing and merging this. I'm excited already :trumpet:

Reply:
Looking into the final bits myself right now!

Reply:
Ooops, sorry, I forgot to change the name.

Reply:
Okay, done. 👍🏻